src/main/scala/io/qbeast/spark/table/IndexedTable.scala

```diff
@@ -165,7 +179,7 @@ private[table] class IndexedTableImpl(
           })
     }
 
-    isNewCubeSize || isNewSpace
+    isNewCubeSize || isNewFileSize || isNewSpace
```

Comment:
Do we have a need to create a new Revision if the desiredFileSize change?

Reply:
Honestly I do not know, the approach was borrowed from the preferred cube size.